mozilla/testpilot-containers
# for #498: security onboarding panels and logic

Not quite ready for merge yet. I need the image for the 3rd security onboarding panel.

To test:
* `shield run . control -- --binary Nightly` should show the existing onboarding panels
* `shield run . securityOnboarding -- --binary Nightly` should show the new security onboarding panels
